src/pages/Overview/OverviewPage.tsx

```diff
@@ -620,6 +620,37 @@ export class OverviewPage extends React.Component<OverviewProps, State> {
           namespaceActions.push(enableAction);
         }
       }
+      if (serverConfig.kialiFeatureFlags.istioUpgradeAction) {
+        const upgradeAction = {
+          isGroup: false,
+          isSeparator: false,
+          isDisabled: !canWrite,
+          title: 'Upgrade to Newer Istio',
+          action: (ns: string) => this.onUpgradeDowngradeIstio(ns, serverConfig.kialiFeatureFlags.istioUpgradeRevision)
+        };
+        const downgradeAction = {
+          isGroup: false,
+          isSeparator: false,
+          isDisabled: !canWrite,
+          title: 'Downgrade to Current Installed Istio',
+          action: (ns: string) => this.onUpgradeDowngradeIstio(ns, serverConfig.kialiFeatureFlags.istioCurrentRevision)
+        };
+        if (
+          nsInfo.labels &&
+          ((nsInfo.labels[serverConfig.istioLabels.injectionLabelRev] &&
+            nsInfo.labels[serverConfig.istioLabels.injectionLabelRev] === 'prod') ||
```

Comment:
The 'prod' constant can be configured by the user, so I'd read this from the config coming from the backend.